parser/src/test/java/com/facebook/coresql/parser/sqllogictest/java/SqlLogicTestConfig.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.coresql.parser.sqllogictest.java;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+import com.facebook.coresql.parser.sqllogictest.java.SqlLogicTest.AcceptedRdbms;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import static com.facebook.coresql.parser.sqllogictest.java.SqlLogicTest.AcceptedRdbms.ALL;
+
+public class SqlLogicTestConfig
+{
+    public static final String TEST_FILES_ROOT_PATH = "src/test/java/com/facebook/coresql/parser/sqllogictest/resources/tests";
```

Comment:
Why the paths are not configurable? If there are configurable then they should not be in the config.

Reply:
Made them configurable.

Done